recipe/meta.yaml

```diff
@@ -38,6 +43,8 @@ requirements:
     - scipy
     - six
     - statsmodels
+    - icu  # [linux and py34]
```

Comment:
I believe the failure will happen in all Pythons and not only `PY34`. Maybe not `PY27`, but still it is OK to add the workaround across the board here.